spec/support/vcr.rb

```diff
@@ -0,0 +1,147 @@
+require 'vcr'
+
+# Cerealizes jason bodies as jason instead of string
+class JasonCerealizer
```

Comment:
Is the name on purpose `JasonCerealizer` instead of `JsonSerializer` ?

Reply:
:upside_down_face:

Reply:
fixed

Reply:
😞

Reply:
> :disappointed:

I know... :cry: